### ISSUE TITLE
[feature request] Disable Autocompletion for specific target

### ISSUE THREAD

**Issue by Faba-network** [TiVo/intellij-haxe#99]
Hi,

it would be great if you have an option to ignore some Std libraries packages in the AutoCompletion.

So you if you write an Javascript Applikation and write something like:
var h:File + (hit AutoCompletion) should only valid Javascript classes appear and no Flash classes.

**Comment by as3boyan** [comment]
Hi,

Cross platform projects need completion. Maybe we can set higher priority for completion item if it's "marked" for current target.

For example, if project target is JavaScript, then move completion for "js" package to the top of completion list.
What do you think about it?

But again, this is more like exception for JavaScript classes declaration, like "js.Browser" and "js.html.*" and other.

Or maybe we can change icon or add additional text to completion item, like "Browser (JavaScript)", or something like that to make items which can be used only using currently active target.

Also, it may be tricky to detect if class is exclusive for some target.

**Comment by Faba-network** [comment]
Hmm i thing the perfect solution is when you have an tab in the sdk or module window where you can activate or deactivate the haxe std folders so you can decide if you need php flash java and so on its the result should be the same if you delete e.g. std/php/. Perhapse you can drag and drop the items so you can set the Priority?!

I create a little mockup perhaps it helps
![mockup](https://cloud.githubusercontent.com/assets/10448310/5683546/0e6b6aa6-9829-11e4-9360-d4f6f42d5bd1.jpg)

**Comment by as3boyan** [comment]
Good idea, perhaps I can implement it with checkbox list as you suggested.

**Comment by fzzr-** [comment]
:+1: Good Idea.
